articles/20_ros_with_dds.md

```diff
@@ -236,13 +252,15 @@ Because this conversion is only invoked in conjunction with a more expensive ser
 This does not preclude the option to improve the `.msg` file format with things like default values and optional fields.
 But this is a different trade-off which can be decided later.
 
+
 ### Services and Actions
 
 DDS currently does not have a ratified or implemented standard for request-response style RPC which could be used to implement the concept of services in ROS.
 There is currently an RPC specification being considered for ratification in the OMG DDS working group, and several of the DDS vendors have a draft implementation of the RPC API.
 It is not clear, however, whether this standard will work for actions, but it could at least support non-preemptable version of ROS services.
 ROS 2.0 could either implement services and actions on top of publish-subscribe (this is more feasible in DDS because of their reliable publish-subscribe QoS setting) or it could use the DDS RPC specification once it is finished for services and then build actions on top, again like it is in ROS 1.x.
-Either way actions will be a first class citizen in the ROS 2.0 API and it may be the case that services just become a degenerate type of action.
+Either way actions will be a first class citizen in the ROS 2.0 API and it may be the case that services just become a degenerated type of actions.
+
```

Comment:
`degenerate` is the right tense here. Maybe a clearer statement would be `services just become a degenerate form of actions.`